courses/free-courses-id.md

```diff
@@ -253,7 +253,8 @@
 * [Tutorial Project React.js 2022](https://youtube.com/playlist?list=PL1aMeb5UP_PHUa0RxQDYJYZZNc_h-jE67) - IDStack
 * [Tutorial React JS Bahasa Indonesia](https://www.youtube.com/playlist?list=PLp6BJq2fT_g91yCNCWi_bIe-ng7S7rt6V) - Lampung JS
 * [Tutorial React Native Bahasa Indonesia (Futsal App)](https://www.youtube.com/playlist?list=PLIan8aHxsPj2NeWJew3o86bSptVPXOppa) - Wahidev Academy
-* [Tutorial React Native Indonesia](https://youtube.com/playlist?list=PLU4DS8KR-LJ3SP3PpRb870UoT_0_rjLpV) - Prawito Hudoro
```

Comment:
did you mean to remove this?

Reply:
Oh sorry, I didn't mean to remove this.

Reply:
well, add it back!

Reply:
Done, I had added the list again.

Reply:
Now there's a duplicate

Reply:
Ah okay, now removed the duplicated list.